lib/beaker/host.rb

```diff
@@ -379,6 +379,13 @@ def do_scp_from source, target, options
       return result
     end
 
+    def do_take_snapshot snapshot_name
```

Comment:
I don't think that this belongs in the host code.  A host is pretty much unaware of what hypervisor is running it and I don't want them to be so coupled to their hypervisor.